contents/blog/0-to-hero.md

```diff
@@ -0,0 +1,160 @@
+---
+date: 2024-01-30
+title: "How to get your first 10, 50, 100, 1000 and 100,000 users"
+rootPage: /blog
+category: PostHog news
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+---
+
+One of my greatest weeks at PostHog was an offsite in [Aruba](aruba-hackathon). As much as I love a Caribbean paradise, it wasn't for the reason you expect.
+
+In 2023, we were having a great year in the face of most of [tech struggling](https://www.theatlantic.com/ideas/archive/2023/01/tech-recession-layoffs-google-facebook-microsoft/672798/), and we wanted something to celebrate. We knew there wouldn't be any big external milestones - like a fundraise for example - to break up the year, and we had a _ton_ of stuff we expected our team to build. We therefore wanted to have a bit of a celebration of how things were going. So we increased the offsite budget, went for a fairly fancy hotel in the Caribbean.
+
+As fun as that sounds - a core part of my job is making sure we don't run out of money. Alas.
+
+We turned up and I felt a little *guilty*. The lobby had a _boat_ in it, there was a _private beach_, you could drink _infinite piña coladas_. Add those up, and I knew we were spending $100K more than we _needed_ to.
+
+As any right-minded founder would:
+
+1. I compartmentalized my feelings
+2. I kept refreshing our revenue dashboard
+
+Our growth had been really consistently increasing each month, and it turned out that the month of our offsite was no exception. It turned out that within ~2 days we'd increased our annual recurring revenue enough to pay for the _entire_ trip. I didn't feel guilty at all. 
+
+This so clearly demonstrated to me the efficiency of our business model - the entire team can be relaxing on a beach and making money whilst not working. Passive income baby! The reason this happens is that PostHog is a 100% inbound company - we've consistently put useful and cool stuff online (like our products or our content), and because we've done a remarkable-enough job of those things, we now just need to keep building on these to grow through word of mouth. It means we don't need people constantly cold emailing or calling to find more business.
+
+If you fast forward today, >900 companies a week now sign up to PostHog, from hobbyists to Fortune 500s.
+
+<PUT IN GRAPH OF WEEKLY COMPREHENSIVE ORG SIGNUPS>
+
```

Comment:
I think this intro is really fun, but... as a reader expecting you to talk about "How to get your first 10, 50, 100, 1000 and 100,000 users", I don't get how it ties in. I think it is just setting the tone that you are legit, but maybe just shorten it a bit 😅

Reply:
covered by title suggestions at start